modules/va_forms/app/sidekiq/va_forms/form_builder.rb

```diff
@@ -8,91 +8,124 @@ class FormBuilder
     include Sidekiq::Job
     include SentryLogging
 
-    # @current_retry set by Sidekiq middleware
-    attr_accessor :current_retry
-
-    RETRIES = 7 # Try 7 times over ~46 minutes
-    FORM_BASE_URL = 'https://www.va.gov'
+    FORM_FETCH_ERROR_MESSAGE = 'The form could not be fetched from the url provided.'
     STATSD_KEY_PREFIX = 'api.va_forms.form_builder'
 
-    sidekiq_options(retries: RETRIES)
+    sidekiq_options retry: 7
+
+    sidekiq_retries_exhausted do |msg, error|
+      args = msg['args'] || {}
+      form_name = args['fieldVaFormNumber']
+      row_id = args['fieldVaFormRowId']
+
+      # Ensure that the form is marked "valid_pdf: false" if successive form fetches have failed
+      if error.message == FORM_FETCH_ERROR_MESSAGE
+        form = VAForms::Form.find_by(row_id:)
+        url = VAForms::Form.normalized_form_url(args.dig('fieldVaFormUrl', 'uri'))
+        form_previously_valid = form&.valid_pdf
+        form&.update!(valid_pdf: false, sha256: nil, url:)
+        if form_previously_valid
+          VAForms::Slack::Messenger.new(
+            {
+              class: self.class.name,
+              message: "URL for form_name: #{form_name}, row_id: #{row_id} no longer returns a valid PDF or web page.",
+              form_url: url
+            }
+          ).notify!
+        end
+      end
```

Comment:
Now, when the job exhausts all of its retries, this block is executed. If the error was due to a failed form fetch, the form record is updated to `valid_pdf: false`. If the form was previously valid, Slack is notified.